### PR TITLE
Fix typo in re docs

### PR DESCRIPTION
**Pull request by berkerpeksag**
Reported by Maksym Nikulyak on docs.p.o.

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]

**Review by vstinner — DISMISSED** [review]
Even if the change is correct, I would prefer a link to the book rather than having to scroll 3 pages. I had to search manually in the doc to find it!

File: Doc/library/re.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ operations; boundary conditions between *A* and *B*; or have numbered group
 references.  Thus, complex expressions can easily be constructed from simpler
 primitive expressions like the ones described here.  For details of the theory
 and implementation of regular expressions, consult the Friedl book referenced
-above, or almost any textbook about compiler construction.
+below, or almost any textbook about compiler construction.
```
Comment (louisom) [inline Doc/library/re.rst]: Maybe add a reference to the `seealso` block

```
consult the Friedl book `mastering regular expressions`_ referenced below
```

```
.. _mastering regular expressions:

.. seealso::
```

**Review by louisom — COMMENTED** [review]
Maybe add a reference to the below seealso block

**Comment by brettcannon** [comment]
To try and help move older pull requests forward, we are going through and backfilling 'awaiting' labels on pull requests that are lacking the label. Based on the current reviews, the best we can tell in an automated fashion is that a core developer requested changes to be made to this pull request.

If/when the requested changes have been made, please leave a comment that says, `I have made the requested changes; please review again`. That will trigger a bot to flag this pull request as ready for a follow-up review.

**Pull request merged**